cluster-autoscaler/.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Can you remove this file?

Reply:
Yeah.. My bad. Carelessly add editor file.